zipkin-web/src/test/scala/com/twitter/zipkin/web/HttpZipkinTracerTest.scala

```diff
@@ -0,0 +1,70 @@
+package com.twitter.finagle.zipkin.thrift
+
+import com.squareup.okhttp.mockwebserver.{MockResponse, MockWebServer}
+import com.twitter.finagle.stats.InMemoryStatsReceiver
+import com.twitter.finagle.tracing._
+import com.twitter.util._
+import com.twitter.zipkin.common.Annotation
+import com.twitter.zipkin.conversions.thrift.thriftListToSpans
+import com.twitter.zipkin.{Constants, common}
+import org.junit.{ClassRule, Test}
+import org.scalatest.junit.JUnitSuite
+
+object HttpZipkinTracerTest {
+  // Singleton as the test needs to read the actual port in use
+  val server = new MockWebServer()
+
+  // Scala cannot generate fields with public visibility, so use a def instead.
+  @ClassRule def serverDef = server
+}
+
+class HttpZipkinTracerTest extends JUnitSuite {
```

Comment:
this class was hanging around in my workspace and is on a different commit than the lowercase stuff.